Back out "Back out "[executorch][PR] Add stories ci for qnn""

cccclai opened the pull request:
Summary:
Original commit changeset: 4e0ee9cac90b

Original Phabricator Diff: D61192484

Differential Revision: D61214261

kirklandsign commented on .ci/docker/ubuntu/Dockerfile:
```diff
@@ -82,5 +82,9 @@ COPY --chown=ci-user:ci-user ./arm /opt/arm
 
+ARG QNN_SDK
+COPY --chown=ci-user:ci-user ./.ci/scripts/setup-qnn-deps.sh setup-qnn-deps.sh
+RUN if [ -n "${QNN_SDK}" ]; then mkdir /opt/qualcomm; chown ci-user:ci-user /opt/qualcomm; ./setup-qnn-deps.sh; fi
```
PWD is `.ci/docker`

so maybe ../scripts/setup-qnn-deps.sh ?

kirklandsign replied:
```
COPY --chown=ci-user:ci-user ../scripts/setup-qnn-deps.sh setup-qnn-deps.sh
```

cccclai committed:
Back out "Back out "[executorch][PR] Add stories ci for qnn"" (#4689)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/4689

Original commit changeset: 4e0ee9cac90b

Original Phabricator Diff: D61192484

Reviewed By: larryliu0820, kirklandsign

Differential Revision: D61214261
